torchvision/transforms/transforms.py

```diff
@@ -6,6 +6,7 @@
 from typing import Tuple, List, Optional
 
 import torch
+from numpy import not_equal
```

Comment:
Do we really need to use numpy. It would be better to avoid adding new dependency

Reply:
I think we don't need it. I do not use it in my code and I'm not sure why this is in here.